#159987365 users can view and edit their profile

joeeasy opened the pull request.

DanielAdek committed:
feat(user): User can sign up

-create button component
-create modal component
-create sigup form

[Delivers #159987361]

DanielAdek committed:
feat(user): User can sign up

-create button component
-create modal component
-create sigup form

[Delivers #159987361]

joeeasy committed:
feat(profile): user can view  and edit profile

- add mock for profile page
- create edit profile
- make mock for view profile responsive
- Make mock for edit profile responsive

[finishes #159987365]

DanielAdek committed:
feat(user): User can sign up

-create button component
-create modal component
-create sigup form

[Delivers #159987361]

DanielAdek committed:
feat(user): Register a user

-create button component
-create modal component
-create sigup form
-test sign up component
-test validation function
[Delivers #159987361]

iAmao commented on src/components/HOC/profileHero/index.js:
```diff
@@ -0,0 +1,209 @@
+Modal.defaultStyles.overlay.backgroundColor = 'rgba(17, 17, 17, 0.74)';
+Modal.defaultStyles.overlay.zIndex = '40';
+Modal.setAppElement('#root');
+class ProfileHero extends Component {
```
add empty line before class

iAmao commented on src/components/HOC/profileHero/index.js:
```diff
@@ -0,0 +1,209 @@
+    padding: '0'
+  }
+};
+
```
remove empty line

joeeasy committed:
feat(profile): user can view  and edit profile

- add mock for profile page
- create edit profile
- make mock for view profile responsive
- Make mock for edit profile responsive
- Users can view the  stats of an author
- Users can view the number of articles the authors has created
- Users can view the interest of an author
- Users can view the bio of an author

[finishes #159987365]
